packages/bee-q/src/components/notification/bq-notification.tsx

```diff
@@ -0,0 +1,188 @@
+  // =======================================
+
+  @State() private isHidden = true;
+
+  // Public Property API
+  // ========================
+
+  /** Type of Notification */
+  @Prop({ reflect: true }) type: TNotificationType = 'default';
+
+  /** Set property if you want Notification icon to be shown. */
+  @Prop({ reflect: true }) showIcon: boolean;
+
+  /** Set the subject color if you don't want to be black. Subject color will also apply to Icon color if there is one. */
+  @Prop({ reflect: true }) subjectColor: string;
+
+  /** Set property to false if you want to hide Close icon */
+  @Prop({ reflect: true }) showClose: boolean;
+
+  // Prop lifecycle events
+  // =======================
+  @Watch('type')
+  checkPropValues() {
+    validatePropValue(NOTIFICATION_TYPES, 'default', this.el, 'type');
+  }
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  componentWillLoad() {
+    this.checkPropValues();
+  }
+
+  // Listeners
+  // ==============
+
+  // Public methods API
+  // These methods are exposed on the host element.
+  // Always use two lines.
+  // Public Methods must be async.
+  // Requires JSDocs for public API documentation.
+  // ===============================================
+
+  /**
+   * Trigger function when you want to close Notification
+   */
+  @Method()
+  async hide() {
+    this.hideNotification();
+  }
+
+  /**
+   * Trigger function when you want to show Notification
+   */
+  @Method()
+  async show() {
+    this.showNotification();
+  }
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private getNotificationIconName = () => {
+    if (!this.showIcon) return '';
+
+    const type = this.type;
+    let icon = 'info';
+
+    if (type === 'success') {
+      icon = 'check-circle';
+    }
+
+    if (type === 'error') {
+      icon = 'x-circle';
+    }
+
+    return icon;
+  };
+
+  private getNotificationColor = () => {
+    const type = this.type;
+    const colors = {
+      default: 'ui--inverse-active',
+      success: 'ui--success',
+      info: 'ui--brand',
+      warning: 'ui--warning',
+      error: 'ui--danger',
+    };
+
+    return isDefined(this.subjectColor) ? this.subjectColor : colors[type];
+  };
+
+  private hideNotification = () => {
+    this.isHidden = true;
+  };
+
+  private showNotification = () => {
+    this.isHidden = false;
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    const styles = {
+      ...(this.subjectColor && { color: getColorCSSVariable(this.subjectColor) }),
+    };
+
+    return (
+      <Host style={styles} aria-hidden={this.isHidden} hidden={this.isHidden}>
```

Comment:
For HTML attributes that receive "true" or "false" values we should do something like:

```jsx
<Host
  class={{ '!hidden': !this.isOpen }}
  aria-hidden={!this.isOpen ? 'true' : 'false'}
  hidden={!this.isOpen ? 'true' : 'false'}
>
```